static/js/enketo/widgets/notewidget.js

```diff
@@ -63,6 +69,24 @@ define( function( require, exports, module ) {
         /* jshint unused:false */
     };
 
+    function applyLiveLinkHtml( $el ) {
+        var html = $el.html();
+
+        html = html.replace( /\[([^\]]*)\]\(([^)]*<[^>]*\>[^)]*)\)/gm,
```

Comment:
I read regex as fluently as nepali, so maybe add a comment describing what this does?